docassemble/ALAffidavitOfIndigency/data/questions/affidavit_of_indigency_supplement_body.yml

```diff
@@ -367,6 +367,13 @@ fields:
   - no label: users[0].nonemployment.there_is_another
     datatype: yesnoradio
 ---
+id: any non-employment incomes
+question: |
+  Do you have any sources of income not from employment?
+fields:
+  - no label: users[0].nonemployment.there_are_any
+    datatype: yesnoradio
```

Comment:
Can't see what the old language was here, but maybe: "Do you get any money that is not from a job?"

Perhaps with "You have already told us [about your jobs | that you do not work]".

Don't want to over-complicate this PR if the old language was basically the same though.

Reply:
I got the wording from `users[0].nonemployment.there_is_another`, but I do like your wording better, so I'll change both (in this interview). I'll try the other jobs thing, if it doesn't take too long.